nuxeo-vision-core/src/test/java/org/nuxeo/vision/core/test/TestPictureEventChain.java

```diff
@@ -76,6 +83,25 @@
 
     @Inject
     protected TagService tagService;
+    
+    @Before
+    public void setup() {
+        Framework.getProperties().put("org.nuxeo.vision.google.credential", "/Users/thibaud/Nuxeo-Misc/google-vision-nuxeo-credentials.json");
```

Comment:
Actually, there are 2 commits. You are looking at the first one. The second commit (top of the page ("Showing change from" => "Show all changes") removed the hard coded things.